lib/template_data/intro.md

```diff
@@ -1,11 +1,116 @@
+some new developments we have yet to discover.
+This is an ongoing project, and it will evolve both on its own, and with the constituent
+packages and notebooks that form its substance.
+We hope you will find it useful.
+
+# PySAL
+
+The original design of PySAL was to have a single monolithic library with subcomponents
+that addressed different types of spatial analysis.
+This facilitated the easy installation of the package for end-users.
+Another guiding principle to minimize the complications of the install was the fairly
+restrictive use of dependencies.
+Over time we have come to recognize that the single monolithic architecture of the
+library, while easing installation, had a number of unintended side effects on the
+developers. Many of the features in the library were buried deep in lower level packages
+and the monolithic structure hindered the discoverability of those packages.
+This meant that the developers of those packages were not getting the recognition they
+deserved, which is particularly important in an academic environment where the time
+dedicated to making these contributions was essentially ignored in tenure and promotion
+cases. Moreover, the limited discoverability also impacted end users who were not aware of
+the functionality.
+
+We recently decided to re-factor the library to address these two limitations.
+The refactoring is recasting PySAL as a meta-package that brings together a federation of
+spatial analytical modules, and carries along several advantages.
+Users who may want to focus on, say, spatial econometrics may have no need for all of
+PySAL, so now they can install `spreg` as its own package.
+The refactoring also increases discoverability as `spreg` is its own active stand-alone package,
+and is no longer buried deep inside PySAL. With this increased visibility, adoption
+increases, leading to greater recognition for the developers as well as more feedback from
+users and, ultimately, improvements to the package.
+From a development perspective, the refactoring also increases the speed at which we are
+able to release new functionality in the individual packages.
+The other benefit of this model is that end users who still want everything in the PySAL
+federation can install the meta-package and should notice no difference from their use of
+the monolithic PySAL package.
+In other words, we support two different ways for users to interface with the library:
+users can get everything in one shot through the meta-package, or they could go the a la
+carte route and pick specific packages in mix and match them to support a specialized
+workflow. 
+
+# Organization
+
+The reorganization of PySAL is along four groups of packages that address the certain type
+of spatial analysis:
+
+1. lib
+2. explore
+3. viz 
+4. model
+
+## Core Data Structures
+
+`libpysal` is the core package and it is here where we handle file-io, spatial weights, and
```

Comment:
Should we add links to each library's notebooks (or package pages) when we mention?